### DISCUSSION TITLE
where change change permission if ldap manage auth and autorisation ?

### DISCUSSION THREAD

**Discussion by spham** [rabbitmq/rabbitmq-server#3142] [Questions] [answered]
hi

I'm guess, how it works when ldap is backnd for auth and autorisation

**Comment by michaelklishin** [comment, marked as answer]
When the [LDAP authn/authz backend is used](https://rabbitmq.com/ldap.html), all operations ultimately result in LDAP queries
(caching not considered), so you need to change the LDAP directory (database).

Multiple backends can be used [in combination](https://www.rabbitmq.com/access-control.html), in that case, you may have to update other sources, e.g. the internal RabbitMQ permission database.
